src/Makefile

```diff
@@ -92,90 +107,50 @@ sunwait:
 	@echo `date +%F\ %R:%S` Building sunwait...
 	@$(MAKE) -C sunwait-src
 	@cp sunwait-src/sunwait .
-	@echo `date +%F\ %R:%S` Build of sunwait complete.
+	@echo `date +%F\ %R:%S` Done.
```

Comment:
To be a little more like the previous behavior, could I get you to create some symlinks in the parent directory? Then when I am in the top level directory and type `make` I get some executables that I could run.

Reply:
right now, the src/Makefile copies the binaries up to the ALLSKY directory.  I can change that behavior to be symlinks if you think that's better.

I guess that does ensure the top level is always the latest built binaries in use.